CMakeLists.txt

```diff
@@ -885,6 +887,34 @@ endif()
 
 ######################################################################################
 
+if (ENABLE_LLVM)
```

Comment:
This (SINGLE_LLVM_LIB CMake option) was removed in a0488d6f152670f5de, and STATIC_LLVM was reintroduced.

Reply:
re-removed the block